json/scores.json

```diff
@@ -122,6 +122,56 @@
                ],
```

Comment:
@muskan6769 There is no problem with your json, but there is a problem with piyu007 json data. I think you can fix that. Try fixing it else ask a maintainer.
@gauthamzz